python/pyarrow/_dataset.pyx

```diff
@@ -2397,12 +2396,14 @@ cdef class Scanner(_Weakrefable):
             record batches are overflowing memory then this method can be
             called to reduce their size.
         batch_readahead : int, default 16
-            The number of batches to read ahead in a file. This might not work
-            for all file formats. Increasing this number will increase
```

Comment:
It seems you (accidentally?) lost the "This might not work for all file formats" ?

Reply:
Fixed.